netbox/extras/views.py

```diff
@@ -605,6 +606,12 @@ def post(self, request, module, name):
         if report is None:
             raise Http404
 
+        schedule_at = None
+        form = ReportForm(request.POST)
+
+        if form.is_valid():
+            schedule_at = form.cleaned_data.get("schedule_at")
```

Comment:
This seems like it will run the report even if the form is invalid. We probably want to follow a pattern similar to `ScriptView.post()`, where we fall back to returning a view of the rendered form.